docs/lang/articles/basic/syntax.md

```diff
@@ -36,16 +34,16 @@ def my_kernel():
 my_kernel()
 ```
 
-Kernels should be called from **Python-scope**.
+Kernels should be called from **Python-scope**, nested kernels are not supported.
```

Comment:
```suggestion
Kernels should be called from **Python-scope**. Nested kernels are not supported.
```
Use a period if you have two sentences :-)

Reply:
Good catch!